### PR TITLE
Single header for all user-exposure.

### PR DESCRIPTION
**Pull request by anokfireball**
To simplify the user experience, a single header with an expressive name is needed for all user-exposed functionality. This header also includes version ```#define```s for the API and the openPMD standard.

### CODE REVIEW

**Comment by anokfireball** [comment]
@ax3l Moving the repo to openPMD apparently caused me to lose write access. May I have it back, plz?

**Comment by ax3l** [comment]
@C0nsultant fixed, sorry default defaults! :)

File: include/openPMD.hpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+#define OPENPMDAPI_VERSION_MAJOR 0
+#define OPENPMDAPI_VERSION_MINOR 1
+#define OPENPMDAPI_VERSION_PATCH 0
+
+#define OPENPMD_STANDARD_MAJOR 1
+#define OPENPMD_STANDARD_MINOR 0
+#define OPENPMD_STANDARD_PATCH 1
+
+#include "Series.hpp"
```
Comment (ax3l) [inline include/openPMD.hpp]: missing single empty EOF newline

File: include/openPMD.hpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* Copyright 2017 Fabian Koller
```
Comment (ax3l) [inline include/openPMD.hpp]: it's 2018

**Comment by ax3l** [comment]
@C0nsultant please as with commit messages, keep the github titles to 40 chars in PRs/issues :)

File: README.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Initial backends will include ADIOS and HDF5.
 
 // ...
 
-auto s = openPMD::Series("output_files/");
+auto s = openPMD::Series::read("output_files/data00000%T.h5");
```
Comment (ax3l) [inline README.md]: that's weird somehow... 0-padding shall be allowed in `%T`

Comment (anokfireball) [reply README.md]: Let me get this right: You would be prefer ```data%T.h5```?
That is still valid and obtains the same files.

I guess I tried to be too smart in the introductory example by showing you can load a limited subset of your series.

**Review by ax3l — APPROVED** [review]

**Pull request merged**